Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs

```diff
@@ -18,14 +21,31 @@ public CandidatesController(ICandidateService candidateService)
             _candidateService = candidateService;
         }
 
-        [EnableQuery(AllowedQueryOptions = AllowedQueryOptions.Filter
-            | AllowedQueryOptions.OrderBy | AllowedQueryOptions.Top
-            | AllowedQueryOptions.Skip | AllowedQueryOptions.Count)]
+        [EnableQuery(
+            HandleNullPropagation = HandleNullPropagationOption.False,
+            AllowedQueryOptions = AllowedQueryOptions.Filter | AllowedQueryOptions.OrderBy | AllowedQueryOptions.Top | AllowedQueryOptions.Skip | AllowedQueryOptions.Count)]
         [ODataAuthorize]
         [ODataRoute]
         public IQueryable<CandidateDTO> Get()
         {
             return _candidateService.Get();
         }
+
+        [EnableQuery(
+            HandleNullPropagation = HandleNullPropagationOption.False,
+            AllowedQueryOptions = AllowedQueryOptions.Filter | AllowedQueryOptions.OrderBy | AllowedQueryOptions.Top | AllowedQueryOptions.Skip | AllowedQueryOptions.Count)]
+        [ODataAuthorize]
+        [HttpGet]
+        public IEnumerable<CandidateDTO> ByProject([FromBody] ODataQueryOptions options, [FromQuery] Guid projectId)
```

Comment:
Remove FromBody, it'S not body, query options are from query: e.g. ?$filter=...; QueryOptions should be generic from model/dto to which you apply them